pkgs/development/libraries/portaudio/default.nix

```
@@ -8,7 +8,8 @@
 , AudioToolbox
 , CoreAudio
 , CoreServices
-, Carbon }:
+, Carbon
+, withALSA ? stdenv.hostPlatform.isLinux && !stdenv.isDarwin }:
```

```suggestion
, withALSA ? stdenv.hostPlatform.isLinux && !stdenv.isDarwin
}:
```
Small formatting nitpick :)